azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/setup/components/SetupControlBarViewModel.kt

```diff
@@ -153,12 +179,32 @@ internal class SetupControlBarViewModel(
     ): Boolean {
         return !shouldControlsBeDisabled(callingState) &&
             audioStateOperation != AudioOperationalStatus.PENDING &&
-            setupScreenOptions?.isMicrophoneButtonEnabled != false
+            setupScreenOptions?.microphoneButton?.isEnabled != false
+    }
+
+    private fun shouldAudioDeviceButtonBeEnabled(
+        callingState: CallingState,
+        setupScreenOptions: CallCompositeSetupScreenOptions?
+    ): Boolean {
```

Comment:
nit: Kotlin shorthand

fun something() -> true/false
Type inferrence, single line collapse to statement.

Reply:
I do use `fun something() -> true/false` in simple cases. I find it hard to read if return is more complex.
For instance, `fun something() -> isConditions1 && isCondition2` then I use shorthand. 
But in this case I find it more cluttered and hurting readability.